web/client/localConfig.json

```diff
@@ -221,7 +221,14 @@
                         "alwaysVisible": true
                     }
                 }
-            }, "Home", "FeatureEditor", "WFSDownload",
+            }, "Home",
+            {
+              "name": "FeatureEditor",
+              "cfg": {
+                "allowedRoles": ["ADMIN", "USER"]
```

Comment:
By default it should be allowed to "ADMIN" only, without any configuration. So remove this